pkg/trait/knative_service_test.go

```diff
@@ -127,21 +127,21 @@ func TestKnativeService(t *testing.T) {
 
 	assert.NotNil(t, s)
 
-	spec := s.Spec.RunLatest.Configuration.RevisionTemplate.Spec
+	spec := s.Spec.ConfigurationSpec.GetTemplate().Spec
 
-	assert.Len(t, spec.Container.VolumeMounts, 5)
+	assert.Len(t, spec.Containers[0].VolumeMounts, 5)
```

Comment:
not a something to do in this pr but we may need to add a method to lookup the integration container instead of guessing its index :)

Reply:
Yes, I took the quick road because it's _test ;)